### PR TITLE
[WIP] doc: add using-jenkins doc

### PR DESCRIPTION
**Pull request by rvagg**
total WIP, putting it here because otherwise it's just floating around between my computers being chipped away at

### CODE REVIEW

File: doc/using-jenkins.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## iojs+any-pr+containers
+
+This job can test any branch of any fork of io.js on GitHub within Linux containers. Use this job to run untrusted code if you need quick confirmation that the code is OK at least on Linux.
+
+## iojs+any-pr+multi
+
+This job can test any branch of any fork of io.js on GitHub across the full CI platform set. While we attempt to limit any cross-over between the test machines and the build machines, there may be some (as of writing there is some reuse of machines for ARM builds). Therefore, it is important that Jenkins users have at least basic level of confidence in the code they are submitting to run on this (and any other non-"containers") job. **Please read diffs before submitting code from untrusted users on this job.**
+
+Currently, all slaves create _Release_ (not _Debug_) builds and run the `simple` and `message`
```
Comment (bnoordhuis) [inline doc/using-jenkins.md]: ...tests?

File: doc/using-jenkins.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## iojs+npm+any-pr+multi
+
+This job can run the `test-npm`
```
Comment (bnoordhuis) [inline doc/using-jenkins.md]: ...tests?

**Comment by jbergstroem** [comment]
The main README.md should probably be updated as well, since it's referring to Buildbot.

File: doc/using-jenkins.md (hunk 1 above)
Comment (rvagg) [inline doc/using-jenkins.md]: heh, yes, I literally saved this file mid-thought here

**Comment by rvagg** [comment]
probably going to have to lean on you @jbergstroem and others to get the README buttoned up, it's been stale for a while but I'm spread too thin

File: doc/using-jenkins.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+Using the io.js Jenkins
+=======================
+
+The io.js Jenkins server can be reached at <https://jenkins-iojs.nodesource.com>. io.js core collaborators can be provided with Jenkins login credentials by the io.js Build team upon request.
+
+The Jenkins server is used to test both the **libuv** and **io.js** projects.
+
+The io.js Jenkins has some separate jobs and while everyone who can log in is able to start any of these jobs, users should limit themselves to just the jobs that are within their area of responsibility.
```
Comment (kenperkins) [inline doc/using-jenkins.md]: nit: I find reading "io.js Jenkins" without a "server" suffix to be confusing.

**Comment by orangemocha** [comment]
Useful stuff. I was thinking of doing something similar in the wiki, where edits would be easier. Thoughts on that?

We also might want to cross links from/to https://github.com/nodejs/io.js/blob/master/COLLABORATOR_GUIDE.md#landing-pull-requests

**Comment by jbergstroem** [comment]
I feel this may have lived its course. I suggest we add more info to `COLLABORATOR_GUIDE.md` instead. Close?

**Pull request closed without merging**